.github/workflows/bump-cli.yml

```diff
@@ -0,0 +1,34 @@
+name: Bump CLI version
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/actions/create-pr/action.yml
+  schedule:
+  - cron: 0 0 */14 * * # run every 14 days
+
+jobs:
```

Comment:
Add a `permissions` block to this job or workflow, with `contents: write` and `pull-requests: write` permissions. That should be enough to allow the workflow to create a PR _when run from the parent repo_. It will still not allow the workflow to create a PR when run from a fork, which I think is preferable.

With this change, I think we can avoid changing the setting to grant read+write permissions to workflows on PRs from forks.

Reply:
Oh ok. I did try that but it failed, I thought maybe it still needed that permission - didn't know it was because it was from the fork, it got rejected. That's great, thanks!

Reply:
It still won't work on your fork, which is ok. The `write` permission in the workflow gets downgraded to `read` on forks, thanks to the repo setting you noticed. It's safer to have read-only permissions on forks than trying to reason about whether we are granting the correct set of permissions to contributor PRs from forks.
 
If you're making automation changes in future it may be easier to use a branch on the parent repo (otherwise forks are totally fine).